### PR TITLE
Independent breadcrumbs

### PR DESCRIPTION
**Pull request by soda736**

### CODE REVIEW

File: web/sites/independentreviewofeducation/modules/independentreviewofeducation_breadcrumbs/src/PublicationBreadcrumb.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  protected $request;
+
+  /**
+   * Class constructor.
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, TitleResolverInterface $title_resolver, RequestStack $request) {
+    $this->entityTypeManager = $entity_type_manager;
+    $this->titleResolver = $title_resolver;
+    $this->request = $request;
+
```
Comment (omahm) [inline web/sites/independentreviewofeducation/modules/independentreviewofeducation_breadcrumbs/src/PublicationBreadcrumb.php:68]: Does this blank line pass phpcs?

Comment (soda736) [reply web/sites/independentreviewofeducation/modules/independentreviewofeducation_breadcrumbs/src/PublicationBreadcrumb.php]: I think it did but I have removed it now.

**Review by ooneill547 — APPROVED** [review]

File: web/sites/independentreviewofeducation/modules/independentreviewofeducation_breadcrumbs/src/PublicationBreadcrumb.php [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build(RouteMatchInterface $route_match) {
+    $breadcrumb = new Breadcrumb();
+    $title_resolver = $this->titleResolver->getTitle($this->request->getCurrentRequest(), $route_match->getRouteObject());
+    $links[] = Link::createFromRoute(t('Home'), '<front>');
+    $links[] = Link::fromTextAndUrl(t('Key documents'), Url::fromUri('internal:/key-documents'));
```
Comment (ooneill547) [inline web/sites/independentreviewofeducation/modules/independentreviewofeducation_breadcrumbs/src/PublicationBreadcrumb.php:115]: Just checking this is the path you actually want for publications in this site?

Comment (soda736) [reply web/sites/independentreviewofeducation/modules/independentreviewofeducation_breadcrumbs/src/PublicationBreadcrumb.php:115]: Yes, they aren't going to have a publications listing page. Just random publications added to the key documents page.

**Pull request merged**